src/main/java/chessmaster/storage/Storage.java

```diff
@@ -127,6 +137,11 @@ public ChessTile[][] loadBoard() throws ChessMasterException {
             fileScanner.nextLine();
         }
 
```

Comment:
We should put all the game-related settings on 1 line separated by a delimiter in future.
`PlayerColor | Difficulty | Turn`
But not needed in this PR, for future updates